l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -198,3 +199,36 @@ def __str__(self) -> str:
         table.float_format = ".2"
         table.add_row(values)
         return str(table)
+
+    def get_scene_dataset(self, scene_index: int) -> "ChunkedDataset":
+        """
+        Get a new ChunkedDataset of a single scene. This dataset lives in memory (as np.ndarray)
```

Comment:
can we use follow SWE excellency to use same doc style as in jadoo? 
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

Reply:
ah yeah I forgot to change from google to sphinx